### ISSUE TITLE
function with reserved keyword generated

### ISSUE THREAD

**Issue by tiagomanczak** [rust-embedded/svd2rust#585]
A specific SFR of the microcontroller has a field which can assume a value descrived by the name DYN.
When generation the functions for the field writer for the variant DYN svd2rust generated the following code

```rust
pub fn dyn(self) -> &'a mut W {
        self.variant(PAD_FRC_A::DYN)
    }
```

So during the build I got 
```console
error: expected identifier, found keyword `dyn`
```

Had to scape manually the function name.
Please see [raw_identifiers](https://doc.rust-lang.org/rust-by-example/compatibility/raw_identifiers.html) for more information.

```rust
pub fn r#dyn(self) -> &'a mut W {
        self.variant(PAD_FRC_A::DYN)
    }
```

Would it be possible to include this as a feature to escape automatically identifiers which match rust's keywords?

**Comment by Emilgardis** [comment]
we already do this, but dyn is not added to the list

https://github.com/rust-embedded/svd2rust/blob/7264731e4796c3b39a302fae89eab36d33a5008d/src/util.rs#L150-L164

feel free to submit a pr :)

**Comment by Emilgardis** [comment]
Awesome :D

Quick tip for you if you did not know already, if the PR had the text

> Fixes #585

This issue would be marked as closed sutomagically, see [this doc](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
